python/cudf/cudf/core/groupby/groupby.py

```diff
@@ -780,6 +784,130 @@ def median(self):
         """Get the column-wise median of the values in each group."""
         return self.agg("median")
 
+    def corr(self, method="pearson", min_periods=1):
```

Comment:
can you also expose this via the docs by adding it in this file: https://github.com/rapidsai/cudf/blob/branch-21.12/docs/cudf/source/api_docs/groupby.rst?

Reply:
It looks like this isn't done yet. Note that there are two `groupby.rst` files, one in the `basics/` directory and one in the `api_docs` directory. The table in the former has been updated, but not the API link in the latter.